src/ralph/lib/mixins/fields.py

```diff
@@ -61,6 +61,63 @@ class NullableCharField(
     pass
 
 
+# NOTE(romcheg): NUMP stands for No Useles Migrations Please.
+#                It is recommended to refrain from checking the other meaning
+#                in UD.
+class NUMPFieldMixIn(object):
+
+    def __init__(self, fields_to_ignore, *args, **kwargs):
+        super(NUMPFieldMixIn, self).__init__(*args, **kwargs)
+        self.fields_to_ignore = fields_to_ignore
+
+    def deconstruct(self):
+        name, path, args, kwargs = super(NUMPFieldMixIn, self).deconstruct()
+
+        if not self.__class__.__mro__.index(NUMPFieldMixIn) > 1:
```

Comment:
I think it would be good to add comment why we need to check this.

Reply:
Done